easybuild/easyblocks/c/cuda.py

```diff
@@ -143,10 +145,22 @@ def make_module_req_guess(self):
 
         guesses = super(EB_CUDA, self).make_module_req_guess()
 
+        # The dirs should be in the order ['open64/bin', 'bin']
+        bin_path = []
+        if LooseVersion(self.version) < LooseVersion('7'):
+            bin_path.append('open64/bin')
+        bin_path.append('bin')
```

Comment:
It's OK to include non-existing directories, the existence of the directories will be checked before they're used in the generated module file

Reply:
So you want me to revert to the original non-version-checking code here? For all of the vars?

Reply:
Well, not sure... I guess there's some value in not making EB look for subdirectories that we know are not there.

So let's leave it as is.

You'll need to fix the issue in `sanity_check_step` though.